docs/components/tab-group.md

```diff
@@ -573,4 +573,60 @@ const App = () => (
 );
 ```
 
+### Set Tab With Param
+
+Allow a URL parameter to determine which tab is selected by default.
+
+```html preview
+<sl-tab-group class="tabs">
+  <sl-tab slot="nav" panel="tab-1">Tab 1</sl-tab>
+  <sl-tab slot="nav" panel="tab-2">Tab 2</sl-tab>
+  <sl-tab slot="nav" panel="tab-3">Tab 3</sl-tab>
+
+  <sl-tab-panel name="tab-1">Tab panel 1.</sl-tab-panel>
+  <sl-tab-panel name="tab-2">Tab panel 2.</sl-tab-panel>
+  <sl-tab-panel name="tab-3">Tab panel 3.</sl-tab-panel>
+</sl-tab-group>
+
+<script>
+  const tabGroup = document.querySelector('.tabs');
+
+  // call customElements.whenDefined('sl-tab') as many times as there are tabs to
+  // ensure all tabs exist.
+  Promise.allSettled([
+    customElements.whenDefined('sl-tab'),
```

Comment:
Was this actually necessary? I thought the element definition would only be called once.

Reply:
no, each tab within the tab group is called once, iirc this one was kind of tricky to figure out.